applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h

```diff
@@ -269,9 +270,25 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement : public Ele
         const ProcessInfo&                               rCurrentProcessInfo) const
     {
         const std::size_t number_of_dimensions = GetGeometry().LocalSpaceDimension();
+        unique_ptr<GeoThermalLaw> law;
```

Comment:
Minor suggestion
```suggestion
        std::unique_ptr<GeoThermalLaw> law;
```

Reply:
@rfaasse Thank you for your help and for reviewing. I have made all the changes you suggested. Then let's wait for the others opinion.